augly/video/augmenters/ffmpeg/noise.py

```diff
@@ -1,31 +1,38 @@
 #!/usr/bin/env python3
 # Copyright (c) Facebook, Inc. and its affiliates.
 
-from typing import Tuple, Dict
+from typing import List
 
-from augly.video.augmenters.ffmpeg.base_augmenter import BaseFFMPEGAugmenter
-from ffmpeg.nodes import FilterableStream
+from augly.video.augmenters.ffmpeg.base_augmenter import BaseVidgearFFMPEGAugmenter
 
 
-class VideoAugmenterByNoise(BaseFFMPEGAugmenter):
+class VideoAugmenterByNoise(BaseVidgearFFMPEGAugmenter):
     def __init__(self, level: int):
         assert 0 <= level <= 100, "Level must be a value in the range [0, 100]"
         self.level = level
 
-    def add_augmenter(
-        self, in_stream: FilterableStream, **kwargs
-    ) -> Tuple[FilterableStream, Dict]:
+    def get_command(self, video_path: str, output_path: str) -> List[str]:
         """
         Adds noise to the video
 
-        @param in_stream: the FFMPEG object of the video
+        @param video_path: the path to the video to be augmented
 
-        @returns: a tuple containing the FFMPEG object with the augmentation
-            applied and a dictionary with any output arguments as necessary
+        @param output_path: the path in which the resulting video will be stored.
+
+        @returns: a list of strings containing the CLI FFMPEG command for
+            the augmentation
         """
-        return (
-            in_stream.video.filter("boxblur", **{"lr": 1.2}).filter(
-                "noise", **{"c0s": self.level, "allf": "t"}
-            ),
-            {},
-        )
+        command = [
+            "-y",
+            "-i",
+            video_path,
+            "-vf",
+            "boxblur=lr=1.2," + f"noise=c0s={self.level}:allf=t",
```

Comment:
can we simply write this:
```suggestion
            f"boxblur=lr=1.2,noise=c0s={self.level}:allf=t",
```
or this:
```suggestion
            "boxblur=lr=1.2,",
            f"noise=c0s={self.level}:allf=t",
```

Reply:
definitely make sure it still works :)